### PR TITLE
Beta.2 Releases

### PR DESCRIPTION
**Pull request by holgerd77**
Part of #907 

Second beta release series for the `VM` `v5` and other monorepo libraries following #922 

All libraries except the `Ethash` library get a new release.

(can be best reviewed on a per-commit basis)

### CODE REVIEW

**Comment by holgerd77** [comment]
Ok, this is now ready for review. I now also updated the `README` files along with this and did most of the tiny fixes from #905

File: packages/vm/README.md [hunk 1]
```diff
@@ -122,44 +66,67 @@ For documentation on `VM` instantiation, exposed API and emitted `events` see ge
...
+- `chainstart` (a.k.a. Frontier) (`v5.0.0`+)
+- `homestead` (`v5.0.0`+)
+- `tangerineWhistle` (`v5.0.0`+)
+- `spuriousDragon` (`v5.0.0`+)
+- `byzantium`
+- `constantinople`
+- `petersburg`
+- `istanbul` (`v4.1.1`+)
+- `muirGlacier` (only `mainnet` and `ropsten`) (`v4.1.3`+)
+- `berlin` (`DRAFT`, only `EIP-2315`)
```
Comment (jochem-brouwer) [inline packages/vm/README.md:92]: 2537 is included as well, but does not work in browser

**Review by jochem-brouwer — COMMENTED** [review]
1 nit comment, rest LGTM! 😄

File: packages/vm/README.md (hunk 1 above)
Comment (holgerd77) [reply packages/vm/README.md:92]: But not as part of the `berlin` hardfork but as a standalone `EIP`, so `EIP-2315` is mentioned in the "Supported EIPs" section.

Comment (holgerd77) [reply packages/vm/README.md:92]: (it is still not decided - respectively the last I heard is that it is becoming rather unlikely - that `EIP-2537` will be included in `berlin`, so seems that we've made the correct design decisions here)

Comment (jochem-brouwer) [reply packages/vm/README.md:92]: Ah woops, you are right! Will approve.

**Review by jochem-brouwer — APPROVED** [review]

File: packages/vm/README.md (hunk 1 above)
Comment (jochem-brouwer) [reply packages/vm/README.md:92]: Sad to see that 2537 won't be available in Berlin, but due to the complexity of that EIP I do understand.

Comment (holgerd77) [reply packages/vm/README.md:92]: I am not 100% sure though that this is confirmed information.

**Comment by holgerd77** [comment]
@jochem-brouwer can you give this a short re-approval? I needed to update the branch here since there was some other PR merged in between (I think I would otherwise do an exception and do an admin-merge here, don't want to postpone the releases just for this)

**Comment by holgerd77** [comment]
@jochem-brouwer update: ah, seems this is necessary only on rebases and not on branch updates (which I did here), no no need for re-approval! 😄  Actually a reason to use this branch update feature a bit more often on somewhat uncritical updates.

**Comment by holgerd77** [comment]
Just published the following releases on npm 😄 :

- [@ethereumjs/common v2.0.0-beta.2](https://github.com/ethereumjs/ethereumjs-vm/releases/tag/%40ethereumjs%2Fcommon%402.0.0-beta.2)
- [@ethereumjs/tx v3.0.0-beta.2](https://github.com/ethereumjs/ethereumjs-vm/releases/tag/%40ethereumjs%2Ftx%403.0.0-beta.2)
- [@ethereumjs/block v3.0.0-beta.2](https://github.com/ethereumjs/ethereumjs-vm/releases/tag/%40ethereumjs%2Fblock%403.0.0-beta.2)
- [@ethereumjs/blockchain v5.0.0-beta.2](https://github.com/ethereumjs/ethereumjs-vm/releases/tag/%40ethereumjs%2Fblockchain%405.0.0-beta.2)
- [@ethereumjs/vm v5.0.0-beta.2](https://github.com/ethereumjs/ethereumjs-vm/releases/tag/%40ethereumjs%2Fvm%405.0.0-beta.2)

**Pull request merged**